src/Directory.Build.props

```diff
@@ -11,24 +11,11 @@
     <PackageProjectUrl>https://dotvvm.com/</PackageProjectUrl>
   </PropertyGroup>
 
-  <!-- Set SourceRevisionId to current git commit hash -->
-  <Target Name="GetGitVersion" BeforeTargets="DispatchToInnerBuilds;BeforeBuild">
-    <Exec Command="git rev-parse HEAD"
-          ConsoleToMsBuild="true"
-          ContinueOnError="true"
-          StandardOutputImportance="low" StandardErrorImportance="low">
-      <Output TaskParameter="ConsoleOutput" PropertyName="SourceRevisionId" />
-    </Exec>
-    <PropertyGroup>
-      <RepositoryCommit>$(SourceRevisionId)</RepositoryCommit>
-    </PropertyGroup>
-  </Target>
```

Comment:
This is now handled by the `Microsoft.SourceLink.GitHub` package?

Reply:
Yup. I just tested it and it added
```
<repository type="git" url="https://github.com/riganti/dotvvm.git" commit="f6d6f7a6101ea5adcb15f647b8f6c863ecb1f8ac" />
```
to the nuspec file in the package.

Nuget package explorer confirms this as well:

![image](https://user-images.githubusercontent.com/13067679/162426018-d073ab5a-b1d9-4bb2-87bd-68b2e0152b56.png)